editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1233,7 +1233,7 @@ SpriteFramesEditor::SpriteFramesEditor() {
 	thumbnail_zoom = 1.0f;
 	max_thumbnail_zoom = 8.0f;
 	min_thumbnail_zoom = 0.1f;
-	sheet_zoom = 1.0f;
+	sheet_zoom = MAX(1, EDSCALE);
```

Comment:
It might be good to add a comment here, as the reason for doing this may not be exactly obvious at a glance:

```suggestion
	// Default the zoom to match the editor scale, but don't dezoom on editor scales below 100% to prevent pixel art from looking bad.
	sheet_zoom = MAX(1, EDSCALE);
```

Remember to squash the commits together after applying this suggestion. See [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase) for instructions :slightly_smiling_face:

Reply:
I can't figure out where do I squash the commits.

Reply:
See https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase for instructions on how to rebase to squash the commits.